people.md

```diff
@@ -75,10 +77,12 @@ Alphabetical by first name, names are followed by GitHub usernames.
 
 ### Committees, Councils, and Working Groups
 
-- Lucy Jimenez, [@LucyJimenez](https://github.com/LucyJimenez) - DEI Working Group (2022-2024)
-- Rollin Thomas, [@rcthomas](https://github.com/rcthomas) - Community Building Working Group (2022-2024)
-- Steven Silvester, [@blink1073](https://github.com/blink1073) - Executive Council (2022-2024)
-- Jason Weill, [@JasonWeill](https://github.com/JasonWeill) - DEI Working Group (2022-2024)
+- Brian Granger, [@ellisonbg](https://github.com/ellisonbg), Executive Council (2023-2024)
```

Comment:
I recommend splitting out former EC members into their own category.

Reply:
Thanks. I was trying to figure out how much we should separate out the organizations, etc., or just leave the list person-oriented.

Let me make that change and see how that looks.

Reply:
I think that does look a lot better. Thanks for the suggestion!